script/Arcueid/Day 13/13_00_ARC13_3.txt

```diff
@@ -91,48 +95,55 @@ The light from the bright moon descends and illuminates the nighttime school.
 [sha:c1d7b158ff2de1831f4df89cfb9f161cacde180c]{
 -- Page 241, Offset 32332.
 -- 「ロアの状態です。真祖は月齢によってその能力限度を上下させます。彼らは満月に近いほど力を増す。それは死徒であるロア本人も同じです。
-"I'm talking about Roa's condition. The limits of a True Ancestor's abilities fluctuates depending on the moon phase. They become stronger the closer it is to a full moon. The same is true even for a Dead Apostle like Roa.
+"I'm talking about Roa's condition.
+ The extent of a True Ancestor's power waxes and wanes in accordance with the phase of the moon.
+ The fuller the moon gets, the stronger they become.
+ The same is true even for Dead Apostles like Roa.
 }
 [sha:f5417c0e00f6b071bde2f05d21043a2c3fd74172]{
 -- Page 241, Offset 32333.
 -- ……今夜のロアはかぎりなく不死身に近いでしょう。今のわたしの装備では殺しきれないかもしれません」
-...It seems like tonight, Roa is close to true immortality. We might not be able to kill him with the tools we currently have."
+...Tonight, Roa will be brushing up against the borders of true immortality.
+ It's possible that I may not be able to kill him with the equipment I have on hand."
 }
 [sha:2bd2d5f06eec228c1a30813c89421e6e2c28ee9d]{
 -- Page 242, Offset 32334.
 -- ―――限りなく不死身に近い、か。
-―――So he's close to true immortality.
+―――So he's close to true immortality, is he.
```

Comment:
The "is he" at the end expresses more suspicion than the jp does imo. I say drop